src/gpuarray/buffer.h

```diff
@@ -746,10 +746,6 @@ typedef enum _ga_usefl {
   /* If you add a new flag, don't forget to update both
      gpuarray_buffer_{cuda,opencl}.c with the implementation of your flag */
   /**
-   * The kernel is made of PTX code.
-   */
-  GA_USE_PTX =      0x1000,
```

Comment:
Why?  The PTX option was only added because the extcopy kernel was written in PTX.  I don't want people to actually use it if they can avoid it.